lib/ansible/plugins/callback/default.py

```diff
@@ -213,7 +214,11 @@ def _print_task_banner(self, task):
         if task_name is None:
             task_name = task.get_name().strip()
 
-        self._display.banner(u"%s [%s%s]" % (prefix, task_name, args))
+        if (task.check_mode or (task.check_mode is None and context.CLIARGS['check'])) and self.check_mode_markers:
```

Comment:
dont need to check CLIARGs as task check_mode should already reflect this

Reply:
Yes, indeed this is not needed any more. Fixed.